tests/test_importvalue.py

```diff
@@ -0,0 +1,39 @@
+import unittest
```

Comment:
Other tests have been written using `pytest` which is more powerful and make tests easier to write. Perhaps it would be better to use it or to switch everything to `unittest` so everything is on the same page.

Reply:
It's easier to use unittest when it comes to imported code, since it's more similar to JUnit. pytest import seems broken though, maybe it's missing from the setup? 
I'd go with both since unittest is a standard library so no extra dependencies are involved. Anyway for you the final word!

Reply:
The `pytest` dependency is here, but it's an extra one only for tests (in the `setup.py`). Everything can be easily launch with `tox`. I think pytest is compatible with `unittest` so it should not be a problem. Beside, once tests are written, it's easy to rewrite them ;). I perform that for another project and the conversion was not so painful, so it's probably better to use `unittest` for now if it speedup the port and to change later if required.